### PR TITLE
add packages vcredist and nssm (DCOS_OSS-5592)

### PR DESCRIPTION
**Pull request by AleksanderBelov**
## High-level description

What features does this change enable? What bugs does this change fix?

## Corresponding DC/OS tickets (required)

  - [DCOS_OSS-5592](https://jira.mesosphere.com/browse/DCOS_OSS-5592) JIRA title / short description.

## Related tickets (optional)

  - [DCOS-ID](https://jira.mesosphere.com/browse/DCOS-<number>) JIRA title / short description.

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]
  
@mesosphere-mergebot changelog-not-required
@mesosphere-mergebot request-review @jongiddy 
@mesosphere-mergebot request-review @DominikDary

### CODE REVIEW

**Comment by AleksanderBelov** [comment]
@mesosphere-mergebot changelog-not-required
@mesosphere-mergebot request-review @jongiddy
@mesosphere-mergebot request-review @DominikDary

**Comment by AleksanderBelov** [comment]
@mesosphere-mergebot request-review @jongiddy

**Comment by DominikDary** [comment]
@AleksanderBelov i have seen this build error in the ci system:
[12:55:47][Building packages] Traceback (most recent call last):
[12:55:47][Building packages]   File "/teamcity/work/f76365414248b6de/build/env/bin/release", line 8, in <module>
[12:55:47][Building packages]     sys.exit(main())
[12:55:47][Building packages]   File "/teamcity/work/f76365414248b6de/build/env/lib/python3.5/site-packages/release/__init__.py", line 952, in main
[12:55:47][Building packages]     release_manager.create('testing', options.channel, options.tag, variants)
[12:55:47][Building packages]   File "/teamcity/work/f76365414248b6de/build/env/lib/python3.5/site-packages/release/__init__.py", line 815, in create
[12:55:47][Building packages]     self.__config['options']['cloudformation_s3_url'] + '/' + repository_path, tree_variants)
[12:55:47][Building packages]   File "/teamcity/work/f76365414248b6de/build/env/lib/python3.5/site-packages/release/__init__.py", line 322, in make_stable_artifacts
[12:55:47][Building packages]     all_completes = do_build_packages(cache_repository_url, tree_variants)
[12:55:47][Building packages]   File "/teamcity/work/f76365414248b6de/build/env/lib/python3.5/site-packages/release/__init__.py", line 572, in do_build_packages
[12:55:47][Building packages]     result = pkgpanda.build.build_tree(package_store, True, tree_variants)
[12:55:47][Building packages]   File "/teamcity/work/f76365414248b6de/build/env/lib/python3.5/site-packages/pkgpanda/build/__init__.py", line 714, in build_tree
[12:55:47][Building packages]     package_sets = [package_store.get_package_set(v) for v in tree_variants]
[12:55:47][Building packages]   File "/teamcity/work/f76365414248b6de/build/env/lib/python3.5/site-packages/pkgpanda/build/__init__.py", line 714, in <listcomp>
[12:55:47][Building packages]     package_sets = [package_store.get_package_set(v) for v in tree_variants]
[12:55:47][Building packages]   File "/teamcity/work/f76365414248b6de/build/env/lib/python3.5/site-packages/pkgpanda/build/__init__.py", line 372, in get_package_set
[12:55:47][Building packages]     return PackageSet(variant, TreeInfo(load_config_variant(self._packages_dir, variant, 'treeinfo.json')), self)
[12:55:47][Building packages]   File "/teamcity/work/f76365414248b6de/build/env/lib/python3.5/site-packages/pkgpanda/build/__init__.py", line 187, in __init__
[12:55:47][Building packages]     package_store
[12:55:47][Building packages]   File "/teamcity/work/f76365414248b6de/build/env/lib/python3.5/site-packages/pkgpanda/build/__init__.py", line 213, in package_tuples_with_dependencies
[12:55:47][Building packages]     for require in package_store.get_buildinfo(*package_tuple)['requires']:
[12:55:47][Building packages]   File "/teamcity/work/f76365414248b6de/build/env/lib/python3.5/site-packages/pkgpanda/build/__init__.py", line 335, in get_buildinfo
[12:55:47][Building packages]     return self._packages[(name, variant)]
[12:55:47][Building packages] KeyError: ('nssm', None)
[12:55:47][Building packages] Process exited with code 1
[12:55:47][Step 3/4] Process exited with code 1
[12:55:47][Step 3/4] Step build_teamcity (Command Line) failed
can you please have a look?

**Comment by AleksanderBelov** [comment]
> @AleksanderBelov i have seen this build error in the ci system:
> [12:55:47][Building packages] Traceback (most recent call last):
> [12:55:47][Building packages] File "/teamcity/work/f76365414248b6de/build/env/bin/release", line 8, in
> [12:55:47][Building packages] sys.exit(main())
> [12:55:47][Building packages] File "/teamcity/work/f76365414248b6de/build/env/lib/python3.5/site-packages/release/**init**.py", line 952, in main
> [12:55:47][Building packages] release_manager.create('testing', options.channel, options.tag, variants)
> [12:55:47][Building packages] File "/teamcity/work/f76365414248b6de/build/env/lib/python3.5/site-packages/release/**init**.py", line 815, in create
> [12:55:47][Building packages] self.__config['options']['cloudformation_s3_url'] + '/' + repository_path, tree_variants)
> [12:55:47][Building packages] File "/teamcity/work/f76365414248b6de/build/env/lib/python3.5/site-packages/release/**init**.py", line 322, in make_stable_artifacts
> [12:55:47][Building packages] all_completes = do_build_packages(cache_repository_url, tree_variants)
> [12:55:47][Building packages] File "/teamcity/work/f76365414248b6de/build/env/lib/python3.5/site-packages/release/**init**.py", line 572, in do_build_packages
> [12:55:47][Building packages] result = pkgpanda.build.build_tree(package_store, True, tree_variants)
> [12:55:47][Building packages] File "/teamcity/work/f76365414248b6de/build/env/lib/python3.5/site-packages/pkgpanda/build/**init**.py", line 714, in build_tree
> [12:55:47][Building packages] package_sets = [package_store.get_package_set(v) for v in tree_variants]
> [12:55:47][Building packages] File "/teamcity/work/f76365414248b6de/build/env/lib/python3.5/site-packages/pkgpanda/build/**init**.py", line 714, in
> [12:55:47][Building packages] package_sets = [package_store.get_package_set(v) for v in tree_variants]
> [12:55:47][Building packages] File "/teamcity/work/f76365414248b6de/build/env/lib/python3.5/site-packages/pkgpanda/build/**init**.py", line 372, in get_package_set
> [12:55:47][Building packages] return PackageSet(variant, TreeInfo(load_config_variant(self._packages_dir, variant, 'treeinfo.json')), self)
> [12:55:47][Building packages] File "/teamcity/work/f76365414248b6de/build/env/lib/python3.5/site-packages/pkgpanda/build/**init**.py", line 187, in **init**
> [12:55:47][Building packages] package_store
> [12:55:47][Building packages] File "/teamcity/work/f76365414248b6de/build/env/lib/python3.5/site-packages/pkgpanda/build/**init**.py", line 213, in package_tuples_with_dependencies
> [12:55:47][Building packages] for require in package_store.get_buildinfo(*package_tuple)['requires']:
> [12:55:47][Building packages] File "/teamcity/work/f76365414248b6de/build/env/lib/python3.5/site-packages/pkgpanda/build/**init**.py", line 335, in get_buildinfo
> [12:55:47][Building packages] return self._packages[(name, variant)]
> [12:55:47][Building packages] KeyError: ('nssm', None)
> [12:55:47][Building packages] Process exited with code 1
> [12:55:47][Step 3/4] Process exited with code 1
> [12:55:47][Step 3/4] Step build_teamcity (Command Line) failed
> can you please have a look?

fix at https://github.com/AleksanderBelov/dcos/commit/56676c61fe2a7fc919709117819807aae03e67cf 
(nssm and vcredict added to array "exclude" in files installer.treeinfo.json and treeinfo.json)

File: packages/treeinfo.json [hunk 1]
```diff
@@ -3,7 +3,9 @@
     "dcos-launch",
     "dcos-metrics",
     "erts",
-    "openstackservice"
+    "openstackservice",
+    "vcredist",
+    "nssm"
```
[the remark this answers is not in the document]
Comment (AleksanderBelov) [reply packages/treeinfo.json]: These are components only to build for Windows. We don't have buildinfo.json and build files for them. We need these packages to be ignored when building for Linux. To do this, they are added to the file packages/treeinfo.json to "exclusion" section.

**Pull request merged**